firstmoversadvantage/fma-ruby-on-rails-template
# 181887431 upgrade to bootstrap 5

#### What's this PR do?
- Updates Rails views with Bootstrap 5

#### How should this be manually tested?
- download master.key from keybase if you don't have any and run server
- click through the app in the browser, especially user forms (devise, opt out form, contact form), Confirm that everything works

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/181887431)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
